server/package.json

```diff
@@ -12,7 +12,8 @@
     "express": "^4.16.2",
     "joi": "^17.2.1",
     "morgan": "^1.9.0",
-    "pg": "^7.3.0",
+    "native": "^0.3.3",
```

Comment:
Hi Kyle, 
Not sure if this dependency is used anywhere. To prevent unnecessarily large dependencies, I would recommend removing this dependency if it is not used.

Reply:
removed native dependency